src/librustdoc/html/render.rs

```diff
@@ -2672,9 +2672,14 @@ fn item_static(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
 
 fn item_function(w: &mut fmt::Formatter, cx: &Context, it: &clean::Item,
```

Comment:
could you strip the constness during the generation of `clean::Function`?

Reply:
I was thinking of tricking that way yes.